modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/source/MBSource.java

```diff
@@ -24,9 +24,24 @@
  * 
  * Wrapper around a {@link JSONObject} containing the "sources" in a Mapbox style. Mapbox sources supply data to be shown on the map. The type of
  * source is specified by the "type" property, and must be one of vector, raster, geojson, image, video, canvas.
- * 
+ *  
  * "Layers refer to a source and give it a visual representation. This makes it possible to style the same source in different ways, like differentiating between types of roads in a highways layer."
  * 
+ * <p>Internally we use a wtms end-point to refer to the data source:
+ * <pre>
+ * "us-states": {
+ *    "type": "vector",
+ *    "url": "https://localhost:8080/geoserver/gwc/service/wmts#us:states"
+ * }
+ * </pre>
+ * </p>
+ * This is based on the following request for a single file::
+ * <pre>http://localhost:8080/geoserver/gwc/service/wmts?
+ *   REQUEST=GetTile&SERVICE=WMTS&VERSION=1.0.0&
+ *   LAYER=dfs:ne_110m_coastline&STYLE=&TILEMATRIX=EPSG:4326:2&TILEMATRIXSET=EPSG:4326&
+ *   FORMAT=application/x-protobuf;type=mapbox-vector
+ *   &TILECOL=3&TILEROW=1</pre>
```

Comment:
Some updates, after testing locally:

- The only geowebcache vector tiles examples we have use TMS, so while WMTS might work, no one on suite knows how to use it. We probably need to get in touch with the openlayers team.
- The URL in our examples is `http://localhost:8080/geoserver/gwc/service/tms/1.0.0/topp:states@EPSG%3A4326@pbf/{z}/{x}/{-y}.pbf`
  Maputnik does not parse / support the `-y` (seems to have trouble with the `-` sign.
- Even when it receives a valid tile from GeoWebCache, Maputnik does not display it.

Ultimately, this will requite an extensive investigation to resolve.